WBooks/Library/Cell/LibraryTableViewCell.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  LibraryTableViewCell.swift
+//  WBooks
+//
+//  Created by daniel.f.munoz on 12/08/21.
+//
+
+import UIKit
+
+class LibraryTableViewCell: UITableViewCell {
+    
+    @IBOutlet weak var containerView: UIView!{
+        didSet{
+            containerView.layer.cornerRadius = 10
+            containerView.layer.shadowColor = UIColor.darkGray.cgColor
+            containerView.layer.shadowOffset = CGSize(width: 0.0, height: 0.1)
+            containerView.layer.shadowOpacity = 0.2
+            containerView.layer.shadowRadius = 3
+            containerView.layer.masksToBounds = false
+            
+        }
+    }
+    @IBOutlet weak var bookImage: UIImageView!{
+        didSet{
+            bookImage.contentMode = .scaleAspectFill
+        }
+    }
+    @IBOutlet weak var titleLabel: UILabel!{
+        didSet{
+            titleLabel.font = UIFont(name: "Gill Sans", size: 16)
+            titleLabel.textColor = UIColor.darkGray
+        }
+    }
+    @IBOutlet weak var autorLabel: UILabel!{
+        didSet{
+            autorLabel.font = UIFont(name: "Gill Sans", size: 14)
+            autorLabel.textColor = UIColor.lightGray
+        }
+    }
+    
+    let identifier = "LibraryTableViewCell"
+    let nibName = "LibraryTableViewCell"
```

Comment:
you could make this static (class variables), so you don't need to instanciate an object to get this values

Reply:
Done